src/tpmstream/io/auto/marshal.py

```diff
@@ -12,7 +13,13 @@ def detect_format_and_yield_buffer(buffer, strict=True):
 
     look_ahead = b""
     try:
-        look_ahead += bytes((next(buffer_iter), next(buffer_iter)))
+        look_ahead += bytes((
```

Comment:
This breaks parsing small blobs like `80 01`.

I'd say let's remove the autodetection for now.

Reply:
Ok, fine with me.

Reply:
I've removed this in the new version posted